src/main/java/com/wcc/platform/repository/postgres/component/MentorMapper.java

```diff
@@ -0,0 +1,56 @@
+package com.wcc.platform.repository.postgres.component;
+
+import com.wcc.platform.domain.platform.member.Member;
+import com.wcc.platform.domain.platform.member.ProfileStatus;
+import com.wcc.platform.domain.platform.mentorship.Mentor;
+import com.wcc.platform.repository.postgres.PostgresMemberRepository;
+import com.wcc.platform.repository.postgres.PostgresMenteeSectionRepository;
+import com.wcc.platform.repository.postgres.PostgresSkillsRepository;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+/** Maps database rows to Mentor domain objects. */
+@Component
+@RequiredArgsConstructor
+public class MentorMapper {
+
+  private final PostgresMemberRepository memberRepository;
+  private final PostgresSkillsRepository skillsRepository;
+  private final PostgresMenteeSectionRepository menteeSectionRepository;
+
+  public Mentor mapRowToMentor(final ResultSet rs) throws SQLException {
+    final long mentorId = rs.getLong("mentor_id");
```

Comment:
I am seen this "mentor_id" and "member_id" a lot and I think we should extract to some new file of constants for database columns for example and avoid deal with strings that are hard to maintain, in case we change the column name, we have to check everywhere.